Mu2eG4/inc/Mu2eG4MTRunManager.hh

```diff
@@ -37,6 +37,10 @@ namespace mu2e {
     void initializeMasterRunAction();
     void stopG4();
     void terminateRun();
+    
+    G4bool SetUpAnEvent(G4Event* evt,
+                        long& s1, long& s2, long& s3,
+                        G4bool reseedRequired);
```

Comment:
For my own edification.  Is this a standard G4 function or one that you added?

Reply:
SetUpAnEvent is a virtual member function of the G4MTRunManager.  It is called by the Worker Run Managers once for each event.  I modified the G4 version for our purposes.

Reply:
Thanks.  Once this settles down we should work with the G4 team via Krzysztof to get them to add hooks for us so that we don't need to modify their code.  The way we have it now will be a nuisance to maintain.